requirements-dev.txt

```
@@ -1,3 +1,4 @@
 -r requirements.txt
+mypy==1.8.0; implementation_name=="cpython"
```

Probably not relevant anymore as we should now only install this in linter stage.
```suggestion
mypy==1.8.0
```